packages/markdown/components/Callout/index.jsx

```diff
@@ -13,7 +15,7 @@ const Callout = props => {
         <span className="callout-icon">{icon}</span>
         {children}
       </h3>
-      {(content.length && content) || (!title ? children : '')}
+      {(content && content.length && content) || (!title && children)}
```

Comment:
Think you have an extra content in here

Reply:
Thought you were right, but check again and I believe this is what happens if:

1. I take out the first reference it flips out about not being able `to read "length" of undefined` when the callout has no body text
2. And taking out the second reference obviously doesn't work cause then it just prints the length of the variable, rather than the actual React tree!